examples/astro/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "astro-example",
   "type": "module",
-  "version": "0.0.1",
```

Comment:
Is the `version` even required if we aren't planning on publishing?

Reply:
I actually thought this is what we did for `design-system-tokens`, but I guess it isn't. I was going for consistency there, but it's not consistent.

I do think every package file has to have a version number in order to be valid.